src/_support/Page/Acceptance/Administrator/MediaManagerPage.php

```diff
@@ -35,18 +35,355 @@ class MediaManagerPage extends AdminPage
 	public static $pageTitleText = 'Media';
 
 	/**
-	 * Powered by Image
+	 * Page title of the media manager listing page.
+	 *
+	 * @var    array
+	 * @since  __DEPLOY_VERSION__
+	 */
+	public static $container = ['class' => 'media-container'];
+
+	/**
+	 * Page title of the media manager listing page.
+	 *
+	 * @var    array
+	 * @since  __DEPLOY_VERSION__
+	 */
+	public static $infoBar = ['class' => 'media-infobar'];
+
+	/**
+	 * The media browser items
+	 *
+	 * @var    array
+	 * @since  __DEPLOY_VERSION__
+	 */
+	public static $itemsContainer = ['class' => 'media-browser-items'];
+
+	/**
+	 * The media browser items extra small
+	 *
+	 * @var    array
+	 * @since  __DEPLOY_VERSION__
+	 */
+	public static $itemsContainerExtraSmall = ['class' => 'media-browser-items-xs'];
+
+	/**
+	 * The media browser items small
+	 *
+	 * @var    array
+	 * @since  __DEPLOY_VERSION__
+	 */
+	public static $itemsContainerSmall = ['class' => 'media-browser-items-sm'];
+
+	/**
+	 * The media browser items medium
+	 *
+	 * @var    array
+	 * @since  __DEPLOY_VERSION__
+	 */
+	public static $itemsContainerMedium = ['class' => 'media-browser-items-md'];
+
+	/**
+	 * The media browser items large
+	 *
+	 * @var    array
+	 * @since  __DEPLOY_VERSION__
+	 */
+	public static $itemsContainerLarge = ['class' => 'media-browser-items-lg'];
+
+	/**
+	 * The media browser items extra large
+	 *
+	 * @var    array
+	 * @since  __DEPLOY_VERSION__
+	 */
+	public static $itemsContainerExtraLarge = ['class' => 'media-browser-items-xl'];
+
+	/**
+	 * The media browser items
+	 *
+	 * @var    array
+	 * @since  __DEPLOY_VERSION__
+	 */
+	public static $items = ['class' => 'media-browser-item'];
+
+	/**
+	 * The media browser items selected locator
+	 *
+	 * @var    array
+	 * @since  __DEPLOY_VERSION__
+	 */
+	public static $itemSelected = ['css' => '.media-browser-item.selected'];
+
+	/**
+	 * The media tree
+	 *
+	 * @var    array
+	 * @since  __DEPLOY_VERSION__
+	 */
+	public static $mediaTree = ['class' => 'media-tree'];
+
+	/**
+	 * The media tree
+	 *
+	 * @var    array
+	 * @since  __DEPLOY_VERSION__
+	 */
+	public static $mediaBreadcrumb = ['class' => 'media-breadcrumb'];
+
+	/**
+	 * Button that toggles the info bar
+	 *
+	 * @var    array
+	 * @since  __DEPLOY_VERSION__
+	 */
+	public static $toggleInfoBarButton = ['class' => 'media-toolbar-info'];
+
+	/**
+	 * The hidden file upload field
+	 *
+	 * @var    string
+	 * @since  __DEPLOY_VERSION__
+	 */
+	public static $fileInputField = 'input[name=\'file\']';
+
+	/**
+	 * The create folder button in the toolbar
+	 *
+	 * @var    string
+	 * @since  __DEPLOY_VERSION__
+	 */
+	public static $toolbarCreateFolderButton = '//button[contains(@onclick, \'onClickCreateFolder\')]';
```

Comment:
You can also add this to the Joomla Browser function, `clickToolbarButton`, just a suggestion

Reply:
We use custom buttons in media manager. It does not work with JoomlaBrowser::clickOnToolbarButton